habitat-baselines/habitat_baselines/common/rollout_storage.py

```diff
@@ -239,7 +239,7 @@ def recurrent_generator(
                 batch["advantages"] = advantages[curr_slice]
             batch["recurrent_hidden_states"] = batch[
                 "recurrent_hidden_states"
-            ][0:1]
+            ][0][0:1]
```

Comment:
What is going on with this? This change should be removed.

Reply:
I need to store in the batch the dimension of the recurrent_hidden_state of every agent because for different policies their dimension will be different as well. A FixedPolicy will have dimension 0. I am current storing this as 
['recurrent_hiddent_states'] = (old_recurrent_hidden_states, [dim_agent1, dim_agent2,...]). Another possibility is to have extra fields called `lengths_recurrent_hidden_states` that contain the dimensions. If you prefer that approach, I can update it.